code/modules/mob/living/living_say.dm

```diff
@@ -212,14 +212,13 @@ GLOBAL_LIST_INIT(message_modes_stat_limits, list(
 			succumb()
 		return
 
+	//Get which verb is prefixed to the message before radio but after most modifications
+	message_mods[SAY_MOD_VERB] = say_mod(message, message_mods)
+
 	//This is before anything that sends say a radio message, and after all important message type modifications, so you can scumb in alien chat or something
 	if(saymode && !saymode.handle_message(src, message, language))
 		return
 	var/radio_message = message
```

Comment:
`var/radio_message = message`
I feel this could be removed with the directly following message edit being moved, but I'm unsure given ✨ byond ✨.